mw-webapp/src/styles/_variables.scss

```diff
@@ -41,6 +41,7 @@ $sidebarZIndex: 100;
 $tooltipZIndex: 110;
 $dropdownZIndex: 110;
 $modalZIndex: 120;
+$toolTipInModalZIndex: 130;
```

Comment:
no we have a;ready such token on 41 line

Reply:
Value of 110 is not enough because tooltips are also needed for the modal statistics window, which is at 120.

Reply:
Refactored